procmgr/processmanager.cpp

```diff
@@ -5144,17 +5127,17 @@ int ProcessManager::addModule(oam::DeviceNetworkList devicenetworklist, std::str
                         continue;
                     }
 
-                    string cmd = installDir + "/bin/remote_command.sh " + IPAddr + " " + password + " 'ls' 1  > /tmp/login_test.log";
+					string loginTmp = tmpLogDir + "/login_test.log";
+                    string cmd = installDir + "/bin/remote_command.sh " + IPAddr + " " + password + " 'ls' 1  > " + loginTmp;
```

Comment:
Not sure where we're getting 'password' in this new version but I suspect I wouldn't like it, and it's being passed around in the clear.